docs/README.md

```diff
@@ -1,63 +1,80 @@
 ## Local Setup
+- Follow the steps below to get this running on your local machine
 
-1. In a terminal window:
-
+### 1. Copy .env
+- Copy the `sample.env` file into `.env`. The command below will do that in the terminal if your CWD(current working directory) is the `contribute` folder.
 ```bash
 cp probot/sample.env probot/.env
 ```
+- If you do not want to populate the database with the freeCodeCamp PR's you can [skip to step 6](#6-start-the-program)
+
+### 2. Update .env
+- Put your GitHub username in the `GITHUB_USERNAME` field of the `.env` file
+
+### 3. Obtain GitHub Personal access token
+- While on Github, click your profile icon > `Settings`
+- Then click `Developer settings` > `Personal access tokens`
+- Click `Generate new token`
+- On the next page, give the token a name so you know what it’s for
+- Scroll down to the bottom and click `Generate token`
+- Copy the token string and paste it into the `GITHUB_ACCESS_TOKEN` field in the `.env` file. Note that you will not be able to see this token string on GitHub again
+
+### 4. Create your own GitHub app
+- While on GitHub, click your profile icon > `Settings`
+- Then click `developer settings` > `New GitHub app`
+- Fill in the `name` field with a name of your choice
```

Comment:
```suggestion
- Fill in the `name` field with a name of your choice
- Fill in the `homepage` field with `https://contribute.freecodecamp.org`
```
Does it matter what we put in there?  I wasn't sure, so I went with the contribute url.

Reply:
@RandellDawson suggested to me to use `https://localhost` - that's what I put in there. I didn't realize it can't be blank so we should probably suggest to put something in there.

Reply:
Seems it doesn't matter much, but I did a bit of googling, found probot's [guide](https://probot.github.io/docs/deployment/#create-the-github-app) and they say:
> Homepage URL: the URL to the GitHub repository for your app

For the sake of putting something, how about we say that?

Reply:
so something like.. 

- Fill in the `homepage` field with the URL to the GitHub repository for your app. e.g.`https://github.com/username/contribute`

Reply:
Yeah, that seems reasonable.